kratos/includes/user_provided_linear_master_slave_constraint.h

```diff
@@ -0,0 +1,319 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Aditya Ghantasala
+//
+//
+
+#if !defined(USER_PROVIDED_LINEAR_MASTER_SLAVE_CONSTRAINT_H)
+#define USER_PROVIDED_LINEAR_MASTER_SLAVE_CONSTRAINT_H
+// System includes
+
+// project includes
+#include "includes/define.h"
+#include "includes/master_slave_constraint.h"
+
+namespace Kratos
+{
+/** \brief Constraint * @class MasterSlaveRelation
```

Comment:
I think this is outdated

Reply:
you mean the description in the comment or the format ?